app/libraries/ExperimentUtilities.php

```diff
@@ -1422,7 +1422,25 @@ public static function get_transfer_details($experimentId)
     public static function getQueueDatafromResourceId($crId)
     {
         $resourceObject = Airavata::getComputeResource(Session::get('authz-token'), $crId);
-        return $resourceObject->batchQueues;
+        $queues =  $resourceObject->batchQueues;
+
+        //Defining maximum allowed value for queue resources
+        $maxNodeCount = Config::get('pga_config.airavata')["max-node-count"];
```

Comment:
Need to check whether the config value exists first.  It won't exist for most gateways.